components/compositing/constellation.rs

```diff
@@ -420,104 +411,155 @@ impl<LTF: LayoutTaskFactory, STF: ScriptTaskFactory> Constellation<LTF, STF> {
     }
 
     /// Handles loading pages, navigation, and granting access to the compositor
-    fn handle_request(&mut self, request: ConstellationMsg) -> bool {
+    #[allow(unsafe_code)]
```

Comment:
When I forgot to add the attribute here I get the following error:
```
<std macros>:5:46: 6:28 error: usage of an `unsafe` block
<std macros>:5 let mut $ rx = sel . handle ( & $ rx ) ; ) + unsafe {
<std macros>:6 $ ( $ rx . add (  ) ; ) + } let ret = sel . wait (  ) ; $ (
/home/keith/Workspace/servo/components/compositing/lib.rs:13:9: 13:20 note: lint level defined here
/home/keith/Workspace/servo/components/compositing/lib.rs:13 #![deny(unsafe_code)]
```
which is very non-informative. Maybe it's a bug? Before when the `select!` macro is in `fn rn(&mut self)` above and when I forgot to add the same attribute, rustc was able to point out the correct line number of where the unsafe code was used, instead of seeing <std macros> here.